src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java

```diff
@@ -47,7 +58,7 @@ public AmountContent create(Claim claim) {
             interestContent,
             formatMoney(claim.getClaimData().getFeesPaidInPound()),
             formatMoney(paidAmount),
-            formatMoney(claimAmount
+            formatMoney(settledClaimAmount
```

Comment:
Here we could have logic `admittedAmount != null ? admittedAmount : claimAmount` to make it more readable.